src/game_api/entities_monsters.hpp

```diff
@@ -94,6 +94,14 @@ class NPC : public Monster
     uint32_t padding3;
 };
 
+enum class GHOST_BEHAVIOR : uint8_t
+{
+    ANGRY = 0,
+    SURPRISED = 1,
+    SAD = 3,
```

Comment:
This doesn't match the original comment, I believe it has to be 0, 1, 2, 3

Reply:
Oh yes, this is a typo of writing 3 instead of 2, then just adding one for the last  😅